src/sdk/namespace_cache.js

```diff
@@ -258,15 +259,17 @@ class NamespaceCache {
                             content_type: params.content_type,
                             xattr: object_info_hub.xattr,
                             last_modified_time: (new Date(object_info_hub.create_time)).getTime(),
+                            update_cache_stats: update_cache_stats_hook(params.bucket)
                         };
+
+                        const start = process.hrtime.bigint();
```

Comment:
Can you please rename it across the file to `start_time` or `timestamp`.
I'm a bit worried using "start" in this code where we often use it as a file offset, and we might shadow another variable (lint should error on that, but there are cases where this might miss).